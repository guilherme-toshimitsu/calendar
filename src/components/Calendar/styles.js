import {memo} from 'react';
import styled from 'styled-components';

const WeekdaysDiv = styled.div`
	overflow: hidden;
	padding: 10px;
`;

export {WeekdaysDiv};
