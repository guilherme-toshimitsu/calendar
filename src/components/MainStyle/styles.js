const styles = (theme) => ({
	root: {
		position: 'relative',
		textAlign: 'center',
		minHeight: '100vh',
		minWidth: 320,
		overflow: 'hidden',
		backgroundColor: theme.mainStyle.mainBackgroundColor,
	},
});

export default styles;
