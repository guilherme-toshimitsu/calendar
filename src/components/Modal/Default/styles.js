const styles = () => ({
  root: {
    width: 400,
    height: 400,
  },
});

export default styles;
