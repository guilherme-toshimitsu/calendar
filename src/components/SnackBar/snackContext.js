import { createContext } from 'react';

const SnackBarContext = createContext({
  snackValues: {
    vertical: 'top',
    horizontal: 'right',
    snackbarType: 'success',
    message: 'general.ok',
    autoClose: false,
    onSuccess: () => {},
  },
  openSnackBar: () => {},
});
const SnackBarProvider = SnackBarContext.Provider;
const SnackBarConsumer = SnackBarContext.Consumer;

export default SnackBarContext;
export { SnackBarConsumer, SnackBarProvider, SnackBarContext };
