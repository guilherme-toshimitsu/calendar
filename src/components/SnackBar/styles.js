import green from '@material-ui/core/colors/green';
import amber from '@material-ui/core/colors/amber';
import blue from '@material-ui/core/colors/blue';

const snackBarContentStyles = (theme) => ({
  success: {
    backgroundColor: green[600],
  },
  error: {
    backgroundColor: theme.palette.error.dark,
  },
  info: {
    backgroundColor: blue[600],
  },
  warning: {
    backgroundColor: amber[700],
  },
  icon: {
    fontSize: 20,
  },
  iconVariant: {
    opacity: 0.9,
    marginRight: theme.spacing(),
  },
  message: {
    display: 'flex',
    alignItems: 'center',
    alignText: 'left',
  },
});

const snackBarStyles = (theme) => ({
  root: {
    maxWidth: '70%',
  },
  close: {
    padding: theme.spacing() / 2,
  },
});

export { snackBarStyles, snackBarContentStyles };
