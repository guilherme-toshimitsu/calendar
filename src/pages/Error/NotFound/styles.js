const styles = () => ({
  root: {
    width: '100%',
    height: '100%',
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center',
  },
  notFoundText: {
    fontSize: 24,
    color: 'red',
  },
});

export default styles;
