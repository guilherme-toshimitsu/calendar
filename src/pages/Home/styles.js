const styles = (theme) => ({
  root: {
    maxWidth: 1360,
    width: '70%',
    margin: 'auto',
    padding: 40,
  },
  title: {
    textAlign: 'left',
    fontSize: 24,
    color: theme.mainStyle.mainThemeColor,
    fontStyle: 'bold',
  },
  text: {
    textAlign: 'left',
    fontSize: 16,
    marginTop: 16,
  },
  subContainer: {
    marginTop: 32,
    marginBottom: 32,
  },
});

export default styles;
