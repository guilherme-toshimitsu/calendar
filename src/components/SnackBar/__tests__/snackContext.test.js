import React from 'react';
import { render } from '@testing-library/react';
import { ThemeProvider } from '@material-ui/styles';
import { createMuiTheme } from '@material-ui/core';

import snackContext from '../snackContext';

const theme = {};

describe('<snackContext />', () => {
  test('if render', () => {
    const { asFragment } = render(
      <ThemeProvider theme={createMuiTheme(theme)}>
        <snackContext />
      </ThemeProvider>
    );

    expect(asFragment()).toMatchSnapshot();
  });

  test.todo('if props change');
});
