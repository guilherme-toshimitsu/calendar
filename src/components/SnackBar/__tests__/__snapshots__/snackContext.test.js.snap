// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<snackContext /> if render 1`] = `
<DocumentFragment>
  <snackcontext />
</DocumentFragment>
`;
